esrally/rally.py

```diff
@@ -169,6 +174,51 @@ def positive_number(v):
         default=""
     )
 
+    provision_parser.add_argument(
+        "--team-repository",
+        help="Define the repository from where Rally will load teams and cars (default: default).",
+        default="default")
+    provision_parser.add_argument(
+        "--car",
+        help="Define the car to use. List possible cars with `%s list cars` (default: defaults)." % PROGRAM_NAME,
+        default="defaults")  # optimized for local usage
+    provision_parser.add_argument(
+        "--car-params",
+        help="Define a comma-separated list of key:value pairs that are injected verbatim as variables for the car.",
+        default=""
+    )
+    provision_parser.add_argument(
+        "--distribution-version",
+        help="Define the version of the Elasticsearch distribution to download. "
+             "Check https://www.elastic.co/downloads/elasticsearch for released versions.",
+        default="")
+    provision_parser.add_argument("--node-ids",
+                                  help="ID(s) of nodes to provision",
+                                  nargs='+',
+                                  default=[0])
+    provision_parser.add_argument(
+        "--node-ip",
+        help="IP of node to provision",
+        default="0.0.0.0")
+    provision_parser.add_argument(
+        "--node-ips",
+        help="IPs of discovery nodes in cluster to join",
+        default="127.0.0.1")
+    provision_parser.add_argument(
+        "--node-name",
+        help="name of node to provision",
+        default=None)
+    provision_parser.add_argument(
+        "--docker",
+        help="provision as Docker container",
+        action="store_true",
+        default=False
+    )
+    for p in [start_parser, stop_parser]:
```

Comment:
This is missing support to enable telemetry devices.

Reply:
I think one can just use the telemetry and telemetry-params args on the root parser, like so:
```
rally --telemetry=jit,gc --challenge=test start --trial-id=2a807f77-410f-4c20-8231-1ea083c9cbe6
```